Readme.md

```diff
@@ -6,6 +6,33 @@ subfolder of this repository makes possible to integrate Energy Web DID
 solution (https://energy-web-foundation.gitbook.io/energy-web/foundational-concepts/self-sovereign-identity#decentralized-identifiers-dids)
 into any REST service that requires this kind of user authentication to be added without changing its source code.
 
+## Sequence diagram
+
+```mermaid
+sequenceDiagram
```

Comment:
I think it is better to use the lib name instead of PDA acronym as some main not be familiar with the acronym.
The other participants are needed to keep the same ordering as is there currently.
```suggestion
sequenceDiagram
    participant client
    participant nginx
    participant auth proxy
    participant PDA as passport-did-auth
    participant backend

```